docs/cli-reference.adoc

```diff
@@ -10,12 +10,12 @@ ___________________
   git clone https://github.com/openshift/nodejs-ex && cd nodejs-ex
   odo create nodejs
   odo push
-
+  
```

Comment:
Unnecessary extra space. Remove the same in other places too

Reply:
This is generated